lib/explorer/series.ex

```diff
@@ -5651,6 +5651,31 @@ defmodule Explorer.Series do
   def split(%Series{dtype: dtype}, _by),
     do: dtype_error("split/2", dtype, [:string])
 
+  @doc """
+  Split a string Series into a struct with fields determined by the list of
+  field names provided.  The length of the field names list determines how
+  many fields the resulting struct will have.  If the string cannot be split
+  into that many separate strings, null values will be provided for the
+  remaining fields.
```

Comment:
```suggestion
  Split a string Series into a struct of string `fields`.
  
  The length of the field names list determines how many times the
  string will be split at most. If the string cannot be split into that
  many separate strings, null values will be provided for the
  remaining fields.
```

The first sentence in docs is always a short summary. :)